apps/dashboard/app/views/batch_connect/session_contexts/_file_navigator.html.erb

```diff
@@ -1,4 +1,48 @@
+<%
+content_for :head do
+  javascript_include_tag 'navigator/index', nonce: true
+end
+%>
 
 <%= label attrib.id, "#{field_options[:label]}" %>
 <%= text_field :batch_connect_session_context, attrib.id, class: 'form-control' %>
-<%= button_tag 'Select Path', type: 'button', class: 'btn btn-primary', style: 'margin-top: 15px;', onclick: 'alert("Onclick Works Properly!");' %>
\ No newline at end of file
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#modelNavigator" id='select-nav-path'>
```

Comment:
May as well fix this margin while it's fresh in my mind.  The button is a bit close to the input field.

```suggestion
<button type="button" class="btn btn-primary mt-2" data-toggle="modal" data-target="#modelNavigator" id='select-nav-path'>
```